File: engine/model/src/main/scala/tools/packet/family/FamilyMessage.scala
package tools.packet.family

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class FamilyMessage(private var _type: Int, private var _mesos: Int) extends PacketInput {
  def theType: Int = _type

  def mesos: Int = _mesos

  override def opcode(): SendOpcode = SendOpcode.FAMILY_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/family/FamilyMessage.java
package tools.packet.family;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record FamilyMessage(Integer theType, Integer mesos) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FAMILY_RESULT;
   }
}